server/metrics/metrics.go

```diff
@@ -38,8 +38,11 @@ var (
 )
 
 func GetGlobalTags() map[string]string {
-	// TODO: add version here
-	return map[string]string{}
+	return map[string]string{
+		"service": config.DefaultConfig.Tags.Service,
+		"env":     config.GetEnvironment(),
```

Comment:
We could probably just switch this back to `Tags` to only set it once (in Tags).

Reply:
I changed it.